opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -1972,4 +1972,41 @@ namespace Opm
     }
 
 
+
+
+    template <typename TypeTag>
+    std::vector<double>
+    MultisegmentWell<TypeTag>::
+    getPrimaryVars() const
+    {
+        const int num_seg = this->numberOfSegments();
+        constexpr int numWellEq = MSWEval::numWellEq;
```

Comment:
I didn't pay enough attention whilst reviewing this PR.  Our CI system on the other hand states that this object (`numWellEq`) [shadows](https://ci.opm-project.org/job/opm-simulators/1734/gcc/new/) another name.  We should probably rename the object in this local scope to avoid the name clash.

Reply:
I'll fix it.